test/indexer-test.js

```diff
@@ -795,8 +798,8 @@ describe('Indexer', function() {
           httpPort: ports.node
         });
 
-        assert.equal(node.txindex, undefined);
-        assert.equal(node.addrindex, undefined);
+        assert.equal(node.txindex, null);
+        assert.equal(node.addrindex, null);
```

Comment:
`assert.equal(null, undefined)` is `true` -- should this be switched to `.strictEqual()` ?

Reply:
```
> bsert.equal(null, undefined)
Thrown:
[AssertionError [ERR_ASSERTION]: null strictEqual undefined] {
  type: 'AssertionError',
  name: 'AssertionError [ERR_ASSERTION]',
  code: 'ERR_ASSERTION',
  generatedMessage: true,
  actual: null,
  expected: undefined,
  operator: 'strictEqual'
}

```

Reply:
Ah interesting OK thats a divergence from nodejs `assert` then I think

Reply:
Yeah, I think for the better, it's the common case. Non-strict equals would be a rare exception.